api/README.md

```diff
@@ -0,0 +1,74 @@
+# MongoDB Atlas Versioned SDK
+
+A Go HTTP client for the [MongoDB Atlas API](https://docs.atlas.mongodb.com/api/)
+
+## Usage
+
+### Adding Dependency 
+
+```
+go install go.mongodb.org/atlas
+```
+
+### Using in the code
+
+```go
+mongodbatlas import "go.mongodb.org/atlas/mongodbatlas/api/v1"
+```
+
+Construct a new Atlas client, then use the various services on the client to
+access different parts of the Atlas API. For example:
+
+```go
+	mongodbatlas import "go.mongodb.org/atlas/mongodbatlas/api/v1"
+
+   	apiKey := os.Getenv("MDB_API_KEY")
+	apiSecret := os.Getenv("MDB_API_SECRET")
+
+	// SDK client with authentication
+	sdk := mongodbatlas.NewSDKClientWithCredentials(apiKey, apiSecret)
```

Comment:
as a follow up to this PR but these constructs with discrete options don't scale well, if we want to offer these wrappers let's follow a functional approach like the current client does

https://github.com/mongodb/go-client-mongodb-atlas/blob/44ada01aee87c1ba1a6897b492117863281487d3/mongodbatlas/mongodbatlas.go#L312-L333

Reply:
Great idea!
It will also make things easier to migrate.

Reply:
Added separate jira as adding it will trigger number of extra files changed.

Reply:
agreed